src/hdf5_time_series_storage.jl

```diff
@@ -182,43 +200,78 @@ function remove_time_series!(
     end
 end
 
+function _make_time_array(timestamps, data, columns::Union{Vector{Symbol}, Nothing})
+    if columns === nothing
+        return TimeSeries.TimeArray(timestamps, data)
+    else
+        return TimeSeries.TimeArray(timestamps, data, columns)
+    end
+end
+
 function get_time_series(
     storage::Hdf5TimeSeriesStorage,
-    uuid::UUIDs.UUID;
-    index = 0,
-    len = 0,
-)::TimeSeries.TimeArray
+    uuid::UUIDs.UUID,
+    index::Int,
+    len::Int,
+    count::Int,
```

Comment:
I think these should have default values, as before.  Return all data.  It reduces the burden on higher-level code.